software/base/src/test/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessStreamsTest.java

```diff
@@ -105,8 +105,8 @@ protected String getStreamOrFail(Task<?> task, String streamType) {
             String stdin = getStreamOrFail(subTask, BrooklynTaskTags.STREAM_STDIN);
             String stdout = getStreamOrFail(subTask, BrooklynTaskTags.STREAM_STDOUT);
             String stderr = getStreamOrFail(subTask, BrooklynTaskTags.STREAM_STDERR);
-            String env = getStreamOrFail(subTask, BrooklynTaskTags.STREAM_ENV);
-            String msg = "stdin="+stdin+"; stdout="+stdout+"; stderr="+stderr+"; env="+env;
+//            String env = getStreamOrFail(subTask, BrooklynTaskTags.STREAM_ENV);
```

Comment:
I strongly dislike commented out that has no comment to say why it's commented out and why it might be useful in the future (versus it being deleted).